pyartcd/pyartcd/pipelines/gen_assembly_lp.py

```
@@ -0,0 +1,497 @@
+        build_data = GitRepository(build_data_path, dry_run=self.runtime.dry_run)
+        await build_data.setup(self.data_path)
+        await build_data.fetch_switch_branch(self.group)
+
+        releases_yaml_path = build_data_path / "releases.yml"
+        if not releases_yaml_path.exists():
+            raise click.ClickException(
+                f"releases.yml not found in {self.data_path} on branch {self.group}"
+            )
+
+        releases_config = yaml.load(releases_yaml_path)
+        releases_model = Model(dict_to_model=releases_config)
```

Ruff (0.15.15)

[error] Undefined name `Model`

(F821)